app-cdr/cdrtools/cdrtools-3.01_alpha17-r1.ebuild

```diff
@@ -0,0 +1,172 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fcaps multilib toolchain-funcs flag-o-matic gnuconfig
+
+MY_P="${P/_alpha/a}"
```

Comment:
Thanks for the suggestion. I looked at that, but versionator doesn't really offer a find and replace string option (the bash substitution only changes 'alpha' to 'a' for downloading from SRC_URI. Not ideal, I agree, but that was there when I got here) :)